src/core/public/chrome/ui/header/header.tsx

```diff
@@ -447,6 +448,8 @@ export function Header({
         </EuiHeaderSection>
         <EuiHeaderSection side="right">{renderRightControls()}</EuiHeaderSection>
       </EuiHeader>
+      <div id="applicationHeaderFilterBar" />
+      <EuiHorizontalRule margin="none" />
```

Comment:
guessing this replaces the border? does this only take up 1px height?

Reply:
yes, otherwise it will be 2 rows and a line separator between the header and the filter bar, which looks really bad and not align with the mockups

Reply:
![image](https://github.com/user-attachments/assets/47efa962-8f03-4325-a668-a5e37c4405e5)

Reply:
Do we use horizontal rules for this elsewhere? If so, that's fine, otherwise, think this should probably be a css border based on the presence of siblings or last of operator.

Reply:
sure, switched back to use css bottom
![image](https://github.com/user-attachments/assets/155588b3-da4f-40b9-bb13-f96d51269005)